selfdrive/car/honda/hondacan.py

```diff
@@ -53,7 +53,7 @@ def create_acc_commands(packer, enabled, active, accel, gas, idx, stopping, star
   accel_command = accel if active else 0
   braking = 1 if active and accel < min_gas_accel else 0
   standstill = 1 if active and stopping else 0
-  standstill_release = 1 if active and starting else 0
+  standstill_release = 1 if active else 0
```

Comment:
Maybe change to pid? It looks like both standstill and standstill_release will be 1 when stopped. But no clue how Honda works here

Reply:
Hmm, yeah changed to not stopping, not sure how it behaves. Will have to test before we merge.